main.go

```diff
@@ -186,7 +236,7 @@ func main() {
 					Longitude: *event.VehiclePosition.Longitude,
 				}
 
-				_ = goakt.Tell(ctx, pid, command)
+				_ = goakt.NoSender.SendAsync(ctx, *vid, command)
```

Comment:
@Tochemey what is the right way to make SendAsync() call here? The sender is the webserver itself, not another actor.

I get the following error now

<img width="1616" alt="image" src="https://github.com/user-attachments/assets/aaabc698-02e9-48f5-9cc9-7376a815532e" />

Reply:
@sdil You need to cache the webserver PID to use it as a sender given the recipient actor name